nixos/modules/services/x11/window-managers/stumpwm.nix

```diff
@@ -15,10 +15,23 @@ in
     services.xserver.windowManager.session = singleton {
       name = "stumpwm";
       start = ''
+        # Set GTK_DATA_PREFIX so that GTK+ can find the themes
+        export GTK_DATA_PREFIX=${config.system.path}
+        # find theme engines
+        export GTK_PATH=${config.system.path}/lib/gtk-3.0:${config.system.path}/lib/gtk-2.0
+        # find the cursor themes
+        export XCURSOR_PATH = ~/.icons:${config.system.path}/share/icons
+
         ${pkgs.stumpwm}/bin/stumpwm &
         waitPID=$!
       '';
     };
-    environment.systemPackages = [ pkgs.stumpwm ];
+
+    environment.systemPackages = [
+      pkgs.stumpwm
+      pkgs.gtk # To get GTK+'s themes.
```

Comment:
is gtk optional here? Isn't that the responsibility of the user to decide what optional feature to install? What about an option `withGtkTheme` or similar?